.ebextensions/default.config

```diff
@@ -0,0 +1,13 @@
+option_settings:
+  aws:elasticbeanstalk:application:environment:
+    DATABASE_USERNAME: '{{resolve:ssm:DATABASE_USERNAME:1}}'
```

Comment:
I'm wondering, is there a way to automatically set up an `.env` file for EB instead of doing it manually like we did before? Not sure how we'd get our db passwords/usernames in there tho...

Reply:
there is but we'd have to commit it, for this reason I think AWS SSM is a better option